Makefile

```diff
@@ -39,57 +43,60 @@ functional: ginkgo
 ifeq (${RUNTIME},)
 	$(error RUNTIME is not set)
 else
-	./ginkgo -v functional/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
-	bash sanity/check_sanity.sh
+	$(QUIET_TEST) ./ginkgo -v functional/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
+	$(QUIET_TEST) bash sanity/check_sanity.sh
 endif
 
 docker: ginkgo
 ifeq ($(RUNTIME),)
 	$(error RUNTIME is not set)
 else
-	./ginkgo -v -focus "${FOCUS}" -skip "${SKIP}" ./integration/docker/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
-	bash sanity/check_sanity.sh
+	# We do not quiet this test, or we get no output for long enough that the CI
+	# systems will presume we have hung, and fail.
+	./ginkgo --succinct --trace -focus "${FOCUS}" -skip "${SKIP}" ./integration/docker/ -- -runtime=${RUNTIME} -timeout=${TIMEOUT}
+	$(QUIET_TEST) bash sanity/check_sanity.sh
 endif
 
 crio:
-	bash .ci/install_bats.sh
-	RUNTIME=${RUNTIME} ./integration/cri-o/cri-o.sh
+	$(QUIET_TEST) bash .ci/install_bats.sh
+	# We do not quiet this test, or we get no output for long enough that the CI
+	# systems will presume we have hung, and fail.
+	bash -c "RUNTIME=${RUNTIME} && ./integration/cri-o/cri-o.sh"
 
 docker-compose:
-	bash .ci/install_bats.sh
-	cd integration/docker-compose && \
-	bats docker-compose.bats
+	$(QUIET_TEST) bash .ci/install_bats.sh
+	$(QUIET_TEST) bash -c "cd integration/docker-compose && bats docker-compose.bats"
 
 docker-stability:
-	systemctl is-active --quiet docker || sudo systemctl start docker
-	cd integration/stability && \
-	export ITERATIONS=2 && export MAX_CONTAINERS=20 && chronic ./soak_parallel_rm.sh
+	$(QUIET_TEST) systemctl is-active --quiet docker || sudo systemctl start docker
```

Comment:
for really muting this, you'd need `$(QUIET_TEST)` on both sides of `||`. But I am fine with how it is now :)